atmosphere/settings/__init__.py

```diff
@@ -41,16 +41,14 @@
 ALLOWED_HOSTS = [unicode(SERVER_URL.replace('https://', ''))]
 
 # NOTE: first admin will be sender of atmo emails.
-ADMINS = (
-    ('AT LEAST ONE ADMIN REQUIRED', 'sends-email@if-debug-false.com'),
-)
+ADMIN = ('AT LEAST ONE ADMIN REQUIRED', 'sends-email@if-debug-false.com')
 
 # Set this value to be different if you want to separate where 'support' related email is sent.
-ATMO_SUPPORT = ADMINS
-ATMO_DAEMON = (("Atmosphere Daemon", "atmo-alerts@iplantcollaborative.org"),)
+ATMO_SUPPORT = ADMIN
+ATMO_DAEMON = ("Atmosphere Daemon", "atmo-alerts@iplantcollaborative.org")
 
 # Django uses this one..
-MANAGERS = ADMINS
+MANAGERS = ADMIN
```

Comment:
Can you look at what `MANAGERS` is used for, this seems unlikely to work if it previously expected an iterable

Reply:
Looks like [`MANAGERS` expects a list](https://docs.djangoproject.com/en/1.9/ref/settings/#std:setting-MANAGERS). Also, this section of the documentation mentions that `ADMINS` is also a variable that django expects to be a list. However, it does not seem like either are being used by django unless `ADMINS` are getting emails for all exceptions.

Reply:
Also, Atmosphere defines `ADMINS` as a list of lists when Django expects a list of tuples so this is more evidence that Django hasn't been using it

Reply:
Yeah both are iterable is the important bit

Reply:
That doc is pretty specific, if we don't use that middleware, we should remove `MANAGERS`

Reply:
I just read the doc for `ADMINS` seems like the right way to go. Basically all ADMINS get notified, and the `EMAIL_HOST_USER` sends the emails. Our app will be all the simpler if we can just use Django conventions. 

What do you think about keeping `ADMINS` a list of tuples, and sending admin emails as `settings.EMAIL_HOST_USER`.

Reply:
I like that 👍